docs/DeveloperGuide.md

```diff
@@ -420,38 +481,75 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The provided student's name is invalid/cannot be found.
 
     * 3a1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
 
-* 3b. Wrong format for deadline was given.
+* 3b. Wrong format for task deadline was provided.
 
     * 3b1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
 
 * 3c. No task description was given.
 
     * 3c1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+**Use case: View all tasks for all students**
+
+**MSS**
+1.  Tuition teacher requests to view all tasks for all students.
+2.  EduManage shows a list of students and their tasks.
+    
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+**Use case: Delete tasks for a student**
+
+**MSS**
+
+1.  Tuition teacher requests to view all tasks for all students.
+2.  EduManage shows a list of students and their tasks.
+3.  Tuition teacher requests to delete a task from a specific student in the list by providing student's name and task index.
+4.  EduManage updates the student’s outstanding tasks list and indicates success.
```

Comment:
EduManage deletes the task from the student’s task list and displays a success message.

Use "delete" consistently throughout and avoid the term "update" in this context to make actions clear.